app/keepers/keepers.go

```diff
@@ -97,6 +95,8 @@ import (
 	txfeestypes "github.com/osmosis-labs/osmosis/v14/x/txfees/types"
 	valsetpref "github.com/osmosis-labs/osmosis/v14/x/valset-pref"
 	valsetpreftypes "github.com/osmosis-labs/osmosis/v14/x/valset-pref/types"
+	epochskeeper "github.com/osmosis-labs/osmosis/x/epochs/v14/keeper"
+	epochstypes "github.com/osmosis-labs/osmosis/x/epochs/v14/types"
```

Comment:
IMO it feels like we should be starting versioning starting v1 as it is now a separate module, or else we would have to update the epoch's versioning everytime we update the main versioning, open to other ppl's thoughts

Reply:
yea, versioning I am open to editing.. There were some problems with previous PR's giving import problems and not being able to find some package in some cosmos-version. so the main reason for the PR is to segregate both